.gitlab-ci.yml

```diff
@@ -234,6 +233,12 @@ windows32:
   except:
     - /^pr-.*$/
 
+pkg:dune-release:
```

Comment:
What's this `pkg` prefix?

Reply:
We took that prefix for package manager dependent builds. We have NiX, then this one in "release" will build an opam package (and maybe upload it).

An important consideration of the release mode tho is that is uses stronger optimizations, so maybe the flambda version should be built like that. I dunno.l, lots of room here.